.goreleaser.yml

```diff
@@ -11,13 +11,15 @@ builds:
       - freebsd
     goarch:
       - amd64
+      - arm64
 archives:
-  - replacements:
-      darwin: Darwin
-      linux: Linux
-      windows: Windows
-      386: i386
-      amd64: x86_64
```

Comment:
Was getting a warning `DEPRECATED: `archives.replacements` should not be used anymore, check https://goreleaser.com/deprecations#archivesreplacements for more info`, so updated to use `name_template` instead

https://app.circleci.com/pipelines/github/grafana/carbon-relay-ng/314/workflows/c8068722-1e83-4cab-83d1-6e5aea53b09c/jobs/2863/parallel-runs/0/steps/0-102